src/pages/Search/SearchPageBottomTab.tsx

```diff
@@ -78,10 +68,9 @@ function SearchPageBottomTab() {
                 )}
                 {isSmallScreenWidth && (
                     <Search
+                        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                        queryJSON={queryJSON!}
```

Comment:
Instead of using the `!` operator, just add check of `queryJSON` to line 69 as part of the check for displaying this block